### PR TITLE
LaunchBar: expansion of ~ and added image extensions

### PR DESCRIPTION
**Pull request by kritdass**
Addressed issues discussed in #4319.

### CODE REVIEW

**Comment by elParaguayo** [comment]
Thanks. This includes the changes from your other PR. Do you want to close that one and we just do it all here?

**Comment by elParaguayo** [comment]
OK. Tests pass. Let me test the code on my machine when I get a chance. If all's good then I'll merge.

File: libqtile/widget/launchbar.py [hunk 1]
```diff
@@ -186,17 +197,25 @@ def setup_images(self):
...
+            if ext in img_extensions:
+                self.icons_files[name] = ipath if os.path.isfile(ipath) else None
             else:
                 # try to add the extension
-                self.icons_files[name] = name + ".png" if os.path.isfile(name + ".png") else None
-        else:
-            self.icons_files[name] = getIconPath(name)
+                for extension in img_extensions:
+                    if os.path.isfile(ipath + extension):
+                        self.icon_files[name] = ipath + extension
```
Comment (elParaguayo) [inline libqtile/widget/launchbar.py:215]:
```suggestion
                        self.icons_files[name] = ipath + extension
```

Comment (kritdass) [reply libqtile/widget/launchbar.py]: Done.

**Pull request merged**